console/account/how-to/resolve-abuse.mdx

```diff
@@ -0,0 +1,63 @@
+---
+meta:
+  title: How to resolve abuses and cases
+  description: This page explains how to resolve abuse tickets.
+content:
+  h1: How to resolve abuses and cases
+  paragraph: This page explains how to resolve abuse tickets.
+tags: resolve abuse complaint ticket scaleway
+dates:
+  validation: 2024-02-19
+  posted: 2024-02-19
+categories:
+  - account
+---
+
+Abuse tickets are complaints regarding the use of Scaleway services for illegal behavior or hosting of illegal content. Abuses for the same issue on the same resource within a given timeframe are grouped by case.
+Scaleway’s clients are responsible for taking the appropriate security measures in order to solve the case and prevent any further abuse from their resources in order to avoid the lock of their services.
```

Comment:
```suggestion
Scaleway’s clients are responsible for taking the appropriate security measures in order to solve the case and prevent any further abuse from their resources in order to avoid the locking of their services.
```